jni/patches/cadical-shared.patch

```diff
@@ -1,31 +1,34 @@
 diff --git a/configure b/configure
-index e761d26..43320d2 100755
+index c02b197..ba0c99b 100755
 --- a/configure
 +++ b/configure
-@@ -471,7 +471,11 @@ mobical:
+@@ -474,6 +474,12 @@ mobical:
  	\$(MAKE) -C "\$(CADICALBUILD)" mobical
  update:
  	\$(MAKE) -C "\$(CADICALBUILD)" update
--.PHONY: all cadical clean mobical test update
 +shared:
 +	\$(MAKE) -C "\$(CADICALBUILD)" libcadical.so
++dynamiclib:
++	\$(MAKE) -C "\$(CADICALBUILD)" libcadical.dylib
 +dll:
 +	\$(MAKE) -C "\$(CADICALBUILD)" cadical.dll
-+.PHONY: all cadical clean mobical test update shared dll
+ .PHONY: all cadical clean mobical test update
  EOF
  
- msg "generated '../makefile' as proxy to ..."
 diff --git a/makefile.in b/makefile.in
-index 69b4d22..f2ff9bb 100644
+index 8fd957f..ad422a1 100644
 --- a/makefile.in
 +++ b/makefile.in
-@@ -47,6 +47,12 @@ mobical: mobical.o libcadical.a makefile
+@@ -49,6 +49,15 @@ mobical: mobical.o libcadical.a makefile $(LIBS)
```

Comment:
Hm? Why add `$(LIBS)` for `mobical` target?
Actually, we do not even use mobical. So, if it does not build on macOS, you can just not build it (i.e. instead of `make`, use `make dylib`).

Reply:
This is because I made patch from the last version of cadical sources and there `$(LIBS)` was added for `mobical` target